python/examples/integrations/writers/Writing_to_WhyLabs.ipynb

```diff
@@ -89,7 +89,7 @@
       "Enter your WhyLabs Org ID\n",
       "Enter your WhyLabs Dataset ID\n",
       "Enter your WhyLabs API key\n",
-      "Using API Key ID:  \n"
+      "Using API Key ID:  ygG04qE3gQ\n"
```

Comment:
Is this ok to get hard coded in? Is this something that will change?

Reply:
That's ok because we're only printing the first 10 characters of the key, and not the complete thing!